crates/tauri/Cargo.toml

```diff
@@ -120,9 +120,13 @@ window-vibrancy = "0.6"
 
 # windows
 [target."cfg(windows)".dependencies]
-webview2-com = "0.36"
+webview2-com = { version = "0.36", optional = true }
 window-vibrancy = "0.6"
-windows = { version = "0.60", features = ["Win32_Foundation"] }
+windows = { version = "0.60", features = [
+  "Win32_Foundation",
+  "Win32_UI",
+  "Win32_UI_WindowsAndMessaging",
```

Comment:
what's that for?

Reply:
We used it in https://github.com/tauri-apps/tauri/blob/4bffc326ea57c1f9301699d5237a29bee8a5c58d/crates/tauri/src/app.rs#L1910

`webview2-com` used to enable those 2 features so we didn't have those feature enabled on `windows` which we should